doc/flakes/design.md

```diff
@@ -2,92 +2,80 @@
 
 ## Goals
 
-* To provide Nix repositories with an easy and standard way to
-  reference other Nix repositories.
+* Standard and easy dependency handling in Nix
 
-* To allow such references to be queried and updated automatically.
+* Discoverability: Be able to query and update dependencies automatically
 
-* To provide a replacement for `nix-channel`, `NIX_PATH` and Hydra
-  jobset definitions.
+* To provide a replacement for `nix-channel`, `NIX_PATH` and Hydra jobset
+  definitions
 
-* To enable reproducible, hermetic evaluation of packages and NixOS
-  configurations.
+* Reproducibility: Evaluate packages and NixOS configurations purely by default
 
-Things that we probably won't do in the initial iteration:
+Upcoming but not yet implemented:
 
-* Sophisticated flake versioning, such as the ability to specify
-  version ranges on dependencies.
+* Sophisticated flake versioning, such as the ability to specify version ranges
+  on dependencies.
 
-* A way to specify the types of values provided by a flake. For the
-  most part, flakes can provide arbitrary Nix values, but there will
-  be some standard attribute names (e.g. `packages` must be a set of
-  installable derivations).
+* A way to specify the types of values provided by a flake. For the most part,
+  flakes can provide arbitrary Nix values, but there will be some standard
+  attribute names (e.g. `packages` must be a set of installable derivations).
 
 
 ## Overview
 
-* A flake is (usually) a Git repository that contains a file named
-  `flake.nix` at top-level.
+* A flake is (usually) a Git repository that contains a file named `flake.nix`
+  at top-level
 
-* Flakes *provide* an attribute set of values, such as packages,
-  Nixpkgs overlays, NixOS modules, library functions, Hydra jobs,
-  `nix-shell` definitions, etc.
+* A flake *provides* an attribute set of values, such as packages, Nixpkgs
```

Comment:
Maybe this should be "A flake *outputs*"?

Reply:
I disagree. I think "to provide" is much clearer. I don't understand what a flake "outputs" would really look like.